equinix/resource_network_device_acc_test.go

```diff
@@ -875,16 +879,18 @@ func TestAccNetworkDevice_PaloAlto_Cluster_Self_BYOL(t *testing.T) {
 					testAccNeDeviceStatusAttributes(&primary, ne.DeviceStateProvisioned, ne.DeviceLicenseStateApplied),
 					testAccNeDeviceClusterAttributes(deviceResourceName),
 					testAccNeDeviceClusterNodeAttributes(&primary, context),
-					testAccNetworkACLTemplateExists(clusterAclResourceName, &primaryAcl),
+					testAccNetworkACLTemplateExists(clusterAclResourceName, &wanAcl),
+					testAccNetworkACLTemplateExists(clusterMgmtAclResourceName, &mgmtAcl),
 					resource.TestCheckResourceAttrSet(deviceResourceName, "acl_template_id"),
+					resource.TestCheckResourceAttrSet(deviceResourceName, "mgmt_acl_template_uuid"),
 					testAccNeDeviceACL(clusterAclResourceName, &primary),
 				),
 			},
 			{
 				ResourceName:            deviceResourceName,
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"cluster_details.0.node0.0.license_token", "cluster_details.0.node1.0.license_token"},
+				ImportStateVerifyIgnore: []string{"cluster_details.0.node0.0.license_token", "cluster_details.0.node1.0.license_token", "mgmt_acl_template_uuid"},
```

Comment:
The `mgmt_acl_template_uuid` field is not returned by the API, therefore it is not `d.Set` or supported for import. 👍 

@rling-equinix should we mention that these three fields are not supported for import? 
https://github.com/equinix/terraform-provider-equinix/pull/115/files#diff-d3be87f03d53e46b1cb409535110711814c46716d6a3ec94c3167f842de3b796R260

Reply:
If a user does not define `mgmt_acl_template_uuid` in their HCL `equinix_network_device` resource config and they `terraform import` a remote resource (not created with Terraform, for example), the desired state of "no management ACL" will not be applied to the remote resource.

This would also be the case for Network Edge devices that have had the Management ACL added in the UI.  The Terraform desired state will not be enforced.

Reply:
Nothing to change here - just pointing this out.

Reply:
To remove a Mgmt ACL, a user would have to replace the existing Mgmt ACL with the UUID referring to a template with an empty `InboundRules` list.

Reply:
Perhaps we should be calling `GetDeviceACLDetails` at https://github.com/equinix/terraform-provider-equinix/blob/63f33e2a542a6e8530c11a41e2f8ffd13a434178/equinix/resource_network_device.go#L902

We would need to update `ne-go` to include all the fields of the `DeviceACLDetails` because today the only field exposed in ne-go is `Status`.

Let's create a new issue for this.

Reply:
#116